src/components/map/layers/edit/NewRoutePathLayer.tsx

```diff
@@ -27,27 +27,28 @@ class NewRoutePathLayer extends Component<IRoutePathLayerProps> {
         const routePathLinks = this.props.routePathStore!.routePath!.routePathLinks;
         if (!routePathLinks || routePathLinks.length < 1) return;
 
-        const res = routePathLinks.flatMap((routePathLink: IRoutePathLink, index) => {
-            const nodeToRender = routePathLink.startNode;
-            return [
-                this.renderNode(nodeToRender, index),
-                this.renderLink(routePathLink),
-            ];
+        const res: JSX.Element[] = [];
```

Comment:
Could be ReactNode[] instead.
https://github.com/sw-yx/react-typescript-cheatsheet#useful-react-prop-type-examples

Reply:
yes